src/biome/text/pipeline.py

```diff
@@ -253,9 +253,9 @@ def train(
             allennlp_logger.setLevel(logging.ERROR)
 
     def predict(self, *args, **kwargs) -> Dict[str, numpy.ndarray]:
-        """Predicts over some input data with current state of the model
+        """Returns a prediction given some input data based on the current state of the model
 
-        The accepted input is dynamically calculated from head.input
+        The accepted input is dynamically calculated from `self.head.input`
```

Comment:
can we add a tip/example to check the signature of predict using self.head.input?

Reply:
i saw that the signature is calculated from the `self.inputs` attribute (property), so i simply added a print statement as small example.